test/parallel/test-tick-processor.js

```diff
@@ -0,0 +1,59 @@
+'use strict';
+var fs = require('fs');
+var assert = require('assert');
+var path = require('path');
+var cp = require('child_process');
+var common = require('../common');
+
+common.refreshTmpDir();
+
+var prevDir = process.cwd();
+process.chdir(common.tmpDir);
+var f = cp.execFileSync(process.execPath, ['-prof', '-pe',
+    'function fib(n) { return n < 2 ? n : fib(n-1) + fib(n-2); }; fib(40);']);
+fs.readdir(common.tmpDir, function(err, files) {
+  if (err) {
+    assert.fail(err);
+  }
+  var matches = [];
+  for (var i in files) {
+    if (files[i].length > 8 && files[i].substr(0, 8) === 'isolate-') {
+      matches.push(files[i]);
+    }
+  }
```

Comment:
I'd shorten this to:

```javascript
var matches = fs.readdirSync(common.tmpDir).filter(function(file) {
  return /^isolate-/.test(file);
});
```

(You could get creative with `.filter(/./.test.bind(/^isolate-))` but that's a judgment call on whether it's more readable.)